### ISSUE TITLE
Update call.mozilla.org for Surveillance Campaign 

### ISSUE THREAD

**Issue by valianttry** [MozillaFoundation/movement-building#49]
Congress will vote on surveillance reform by May 21. Phone calls to Congress could have a big impact on how members of Congress vote. Fight215.org is a coalition we are a part of, and they are driving as many calls as possible leading up to May 21st. 

Tasks to complete (screenshots below):
- [x] Update the headline on call.mozilla.org
- [x] Remove the "countdown timer" on call.mozilla.org
- [x] Update the "here's what to say" script text (@valianttry leads)
- [x] Update the recordings (@valianttry leads):
  - [x] Tweak FFTF's greeting to say "on behalf of Mozilla" etc.
  - [ ] Tweak the zip code recording to say "enter your zip code on your phone keypad"
  - [ ] Update the "here's what to say" recording to match our "here's what to say" text on call.mozilla.org
- [x] Replace the current email opt-in Thank You confirmation page with the Thank You page that asks people to call (re-use the Net Neutrality Thank You, no edits needed on the page)
- [ ] Make certain all metrics are correctly configured for tracking
- [x] Make certain the "give us feedback" survey and Google Doc are still functional

![sendto_mozilla_org___2015_fccvote_net_neutrality_thanks](https://cloud.githubusercontent.com/assets/9830371/7508323/64548f04-f433-11e4-8937-d85d9be8376e.png)

![mozilla_firefox](https://cloud.githubusercontent.com/assets/9830371/7508270/8718c632-f432-11e4-8507-294a953ea171.png)

![mozilla_firefox](https://cloud.githubusercontent.com/assets/9830371/7508255/445aecee-f432-11e4-8d33-ea415dd92378.png)

**Comment by edrushka** [comment]
This is super, Andrea. Thanks for the detail. 
I want to clarify a couple things for @ScottDowne 
1) this is currently P2. Getting the interstitial to work on mobile is P1.
2) step one for this is to estimate how much time this will take
3) Scott's P1 for next heartbeat is the MozFest site, so if this is going to take a ton of effort, we'll need discuss whether to proceed

Scott, when you have a moment, can you give us a time estimate, and whether or not this is a realistic P2 for next heartbeat?

**Comment by valianttry** [comment]
Is it possible this can be live by Wed May 12th?

TL;DR: that's when the House votes and the coalition is turning the volume up to 11 on May 11-12. The louder advocates can be before that vote, the more impact we will have on the final bill content.

**Comment by edrushka** [comment]
Do you mean **Tuesday May 12** or Wednesday May 13? Or (oh god, please no) Monday May 11?

**Comment by ScottDowne** [comment]
Front end point of view this is a days work, to get to staging.

Some QA, and Monday is doable, the 12th is comfy.

Tomorrow I'll make an end to end and see if there are any fun surprises.

**Comment by valianttry** [comment]
:clap:  
I'm working on finalizing the call script and then recordings.

**Comment by valianttry** [comment]
Also i prefer to think of :clap: as jazz hands.

**Comment by valianttry** [comment]
@ScottDowne 

Final call script is here (this is approved by policy and ready to go on call.mozilla.org):

I'm calling to urge you to vote YES on the USA FREEDOM Act and put an end to illegal mass surveillance under the Patriot Act. Ending mass surveillance will help restore trust in the Internet and improve security for us all. Now is not the time to delay, please oppose any reauthorization of the Patriot Act without real reforms.

**Comment by valianttry** [comment]
@davidascher or @simonwex I'm checking with Twilio to make sure they're cool comping us again for this campaign. JP handled our Twilio account and credit card setup for net neutrality -- just checking to make sure that card is still good and does @ScottDowne or someone else have access to our profile if needed? Thanks!

**Comment by ScottDowne** [comment]
Staging link: http://scottdowne.github.io/mass-surveillance-call/

Not sure if the Twilio stuff works out of the box or if I need to make changes. Right now it's sending to `https://callcongress.mofoprod.net/create?campaignId=jan14th&userPhone=` I'm guessing the campaign id is wrong and we need to setup a new one, but it might work...

Feedback is sent here: https://docs.google.com/spreadsheets/d/1n7JHP5ftKSI3dWAopOEXkgXcpHQy3oz9kjs9LPMrx-M/edit#gid=1930464889

**Comment by ScottDowne** [comment]
@jbuck cc'ing you, for the Twilio or `https://callcongress.mofoprod.net` bits.

**Comment by valianttry** [comment]
I just pinged Jeff @ Fight for the Future again to see if he's able to help with recordings today. This is looking good!

**Comment by valianttry** [comment]
Jeff replied, he said he'd send over recordings today. (He is PST, FYI)

**Comment by ScottDowne** [comment]
I'm not super sure how to setup a new campaign on `https://callcongress.mofoprod.net`

From what I can tell, I need to setup a new block like this: https://github.com/mozilla/call-congress/blob/master/data/campaigns.yaml#L224-L264

Here is the initial commit that set that up https://github.com/mozilla/call-congress/commit/694311148930b265f69269da16b878cefabcb4c6

Docs for campaign data is found here: https://github.com/fightforthefuture/call-congress

**Comment by ScottDowne** [comment]
Things needing to be done, in this order:
- [x] Create a campaign in: https://github.com/mozilla/call-congress/blob/master/data/campaigns.yaml
- [x] Get the new audio files and link them in the campaign created above
- [x] Update https://callcongress.mofoprod.net with the above updates
- [x] Update https://github.com/ScottDowne/mass-surveillance-call/blob/gh-pages/call-congress.js#L121 with the new campaign id
- [x] Update call.mozilla.org with the above changes
- [x] Update the thank you page with a link to the new call tool.

**Comment by ScottDowne** [comment]
I created a new campaign in a pull request, it needs a review by someone like Jeff and links to the new audio files: https://github.com/mozilla/call-congress/pull/1

**Comment by rubbingalcoholic** [comment]
@ScottDowne left a comment on your campaign.yaml update. You're on the right track, the only requirement to adding a new campaign is editing that file and make sure the audio recordings exist at the referenced URLs.

**Comment by rubbingalcoholic** [comment]
@ScottDowne I am unable to assist with recording the audio though. Please find someone within Mozilla. You can listen to the existing .mp3 files referenced in the campaign.yaml file to understand what language each recording needs to incorporate.

**Comment by edrushka** [comment]
@valianttry We need a voice. Suggestions?

**Comment by valianttry** [comment]
Yikes. Re-recording all of the files is going to be very time consuming. I am not setup here to record (my home office is rather noisy and i don't have a great headset). Do we need to create all new files or just the opening file with the Mozilla intro + caller script?

**Comment by valianttry** [comment]
Confirmed that Twilio will comp our use of their tool for this campaign as well:

On Mon, May 11, 2015 at 12:56 PM, Rob Spectre rspectre@twilio.com wrote:

```
Go get 'em!

On Mon, May 11, 2015 at 10:05 AM, Andrea Wood <andrea@mozillafoundation.org> wrote:

    Hey Rob, just lifting this up in your inbox. We're hoping to launch this tool today/tomorrow if possible. Thanks!


    ---
    Andrea Wood
    Director, Digital Organizing + Fundraising
    Mozilla

    [v] +1 510-922-0181
    @valiant

    ---------- Forwarded message ----------
    From: Andrea Wood <andrea@mozillafoundation.org>
    Date: Fri, May 8, 2015 at 1:24 PM
    Subject: Re: Fwd: Your Twilio account has been suspended
    To: Rob Spectre <rspectre@twilio.com>


    Hey Rob,

    First, thanks to you and Twilio for supporting Mozilla's advocacy. The call tool racked up 7K calls and we won net neutrality -- so booya!

    We're working with FFTF to do another round of calls from call.mozilla.org for the new surveillance campaign. Just checking to see if we have limitations for freebee services before we hit the "go" button on this one. I suspect we may have the same # of calls roughly for this campaign.

    Please know I can find budget for this if need be, but want to be clear on what you guys are comfortable doing for us in the future, thanks SO much for all you've done so far.

    Andrea
```

**Comment by edrushka** [comment]
@ScottDowne @rubbingalcoholic  Do we need to create all new files or just the opening file with the Mozilla intro + caller script?

**Comment by ScottDowne** [comment]
Here is a list of the audio files from the NN campaign:

  msg_intro: https://battleforthenet.s3.amazonaws.com/audio/mozilla_intro.mp3
  msg_intro_confirm: http://battleforthenet.s3.amazonaws.com/audio/jan14th_prompt.mp3
  msg_call_block_intro: https://battleforthenet.s3.amazonaws.com/audio/mozilla_call_block_intro.mp3
  msg_between_thanks: http://battleforthenet.s3.amazonaws.com/audio/jan14th_msg_between_thanks_2.mp3
  msg_final_thanks: http://battleforthenet.s3.amazonaws.com/audio/jan14th_msg_final_thanks.mp3

**Comment by ScottDowne** [comment]
Just msg_intro and msg_final_thanks need to be updated.

Although having a unified voice probably is a good idea.

**Comment by ScottDowne** [comment]
I have a staging link for the updated thank you page: https://sendto.mozilla.org/page/s/section-215-petition-thanks-staging

I can turn that on once call.mozilla.org is ready to go.

**Comment by edrushka** [comment]
I have an audio recorder so I guess I could do it. But I'd need a script. It'll take me FOREVER to transcribe the files.

**Comment by valianttry** [comment]
I'll work on scripts. I'm making an executive decision to just record the opening and closing for time-saving. We'll keep an eye on the feedback survey responses to see if people are concerned about 2 diff. voices.

**Comment by edrushka** [comment]
Do you need me to do the voice? I'll have to dig out my recorder.

**Comment by valianttry** [comment]
Anyone with a headset can use Garageband or iTunes I think -- that might be better? What about asking someone on Webmaker to help out? (Just know you've no doubt got other things to do @edrushka )

**Comment by edrushka** [comment]
ok. well maybe we should get a male voice to match Jeff's clips. Maybe bobby would do it.

**Comment by valianttry** [comment]
@edrushka here are the scripts (already approved by policy). Whoever records just needs a good microphone in a quiet (silent) room -- a recording booth would be best. They should listen to the existing recordings to get the "tempo" (Jeff's originals). Then, they need to record themselves speaking these two passages, saved as .mp3 in 2 separate files:

msg_intro
Hi, and on behalf of Mozilla thank you for making a stand with us to stop mass surveillance and protect privacy. There are only days left before an important vote in Congress, so every call counts. We've made it really easy. During this call, we'll connect you to the offices of your U.S. Senators and Representative. When you're connected, give your name and city so they know you are a constituent. And say: I'm calling to urge you to vote YES on the USA FREEDOM Act and put an  end to mass surveillance under the Patriot Act. Ending mass  surveillance will help restore trust in the Internet and improve  security for us all. Now is not the time to delay, please oppose any re-authorization of the Patriot Act without real reforms.

msg_final_thanks:
Thank you so much for participating. We have a real opportunity to rein in mass surveillance. If you can tell your friends and help spread the word about how they can help too, that will matter a lot. Thanks.

**Comment by valianttry** [comment]
@KevZawacki would you be game to help us with this small, but crucial, recording task? ^^

**Comment by valianttry** [comment]
@KevZawacki actually said he'd "be honored" :clap: :crown:

**Comment by valianttry** [comment]
For some reason we have 2 links to Report Back, but we only need 1 -- can we just remove the text line in favor of the button?

![mozilla_firefox](https://cloud.githubusercontent.com/assets/9830371/7601205/0b075ca8-f8c8-11e4-858e-528af3b4fb8a.png)

**Comment by valianttry** [comment]
Update the text script to match the voice script in our prompts. The text should say:

I'm calling to urge you to vote YES on the USA FREEDOM Act and put an end to mass surveillance under the Patriot Act. Ending mass surveillance will help restore trust in the Internet and improve security for us all. Now is not the time to delay, please oppose any re-authorization of the Patriot Act without real reforms.

**Comment by valianttry** [comment]
Chrome works great. I noticed that the Thank You page still references the FCC Vote. Let's update it, and would it be possible to add a text-only donation link?:
![https___call_mozilla_org__](https://cloud.githubusercontent.com/assets/9830371/7601294/0b626a16-f8c9-11e4-9044-d9fac70f5459.png)

**Comment by ScottDowne** [comment]
Closing
